### PR TITLE
Add Kollaborate Transfer 1.3.0.0

### PR DESCRIPTION
**Pull request by NaanProphet**
Batch uploader application for use with Digital Rebellion's Kollaborate
video editing servers. Version-specific URL deduced.

### CODE REVIEW

File: Casks/kollaborate-transfer.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'kollaborate-transfer' do
+  version '1.3.0.0'
+  sha256 '3fcfd9bab9232133b2a78bcbfc6fecf1b4730adf014996d5bd2fdf57d2e5b734'
+
+  url "http://www.digitalrebellion.com/download/kollabtransfer?version=#{version.no_dots}"
```
Comment (jawshooah) [inline Casks/kollaborate-transfer.rb]: Needs a comment `# digitalrebellion.com is the official download host per the vendor homepage`

**Comment by SirDap** [comment]
Ah thank you! Forgot about that. Coming right up

File: Casks/kollaborate-transfer.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+cask 'kollaborate-transfer' do
+  version '1.3.0.0'
+  sha256 '3fcfd9bab9232133b2a78bcbfc6fecf1b4730adf014996d5bd2fdf57d2e5b734'
+
+  # digitalrebellion.com is the official download host per the vendor homepage
+  url "http://www.digitalrebellion.com/download/kollabtransfer?version=#{version.no_dots}"
+  name 'Kollaborate Transfer'
+  homepage 'https://www.kollaborate.tv/resources'
```
Comment (jawshooah) [inline Casks/kollaborate-transfer.rb]: It's listed as free on the homepage. Why `:commercial`?

File: Casks/kollaborate-transfer.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  license :commercial
```
Comment (jawshooah) [inline Casks/kollaborate-transfer.rb]: It's listed as free on the homepage. Why `:commercial`?

**Comment by SirDap** [comment]
Same reasoning as https://github.com/caskroom/homebrew-cask/pull/17158#issuecomment-172290925

**Comment by SirDap** [comment]
Updated `license`!

**Comment by adidalal** [comment]
Thank you for the contribution. It was merged directly as 66fd4a2361237adb1b866522e1b657dd62e84807 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**